homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -299,22 +316,34 @@
     ATTR_TARGET_HUMIDITY: "target_humidity",
     ATTR_LED_BRIGHTNESS: "led_brightness",
     ATTR_USE_TIME: "use_time",
-    ATTR_HARDWARE_VERSION: "hardware_version",
 }
 
 AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER = {
     **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_HARDWARE_VERSION: "hardware_version",
```

Comment:
Is this available on all humidifiers?

Reply:
After testing - the property is not available for miot humidifier, I received a error about get the attribute.

Reply:
But I hope yes, it was for all, because I just removed it from common and set for previous models.